error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ImmutableMapRules.java

```diff
@@ -430,4 +430,20 @@ ImmutableMap<K, V> after(@Repeated Map.Entry<? extends K, ? extends V> entries)
   // map.entrySet().stream().filter(keyPred).forEach(mapBuilder::put)
   // ->
   // mapBuilder.putAll(Maps.filterKeys(map, pred))
+
+  /**
+   * Prefer {@link ImmutableMap.Builder#put(Object, Object)} over {@link
+   * ImmutableMap.Builder#putAll(Map)} when adding a single key-value pair.
+   */
+  static final class ImmutableMapBuilderPutOverPutAllSingleEntry<K, V> {
```

Comment:
Let's make sure the _XXX_ comment remains at the bottom. Also:
```suggestion
  static final class ImmutableMapBuilderPut<K, V> {
```

Reply:
done